src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import docker
+import inspect
+import json
+import os
+import sys
+import syslog
+import time
+
+from swsscommon import swsscommon
+
+sys.path.append('/usr/share/sonic/scripts/')
+import container_state
+
+state_db = None
+
+# DB field names
+FEATURE_TABLE = "FEATURE"
+SET_OWNER = "set_owner"
+NO_FALLBACK = "no_fallback_to_local"
+
+CURRENT_OWNER = "current_owner"
+UPD_TIMESTAMP = "update_time"
+DOCKER_ID = "container_id"
+REMOTE_STATE = "remote_state"
+VERSION = "version"
+SYSTEM_STATE = "system_state"
```

Comment:
i am not familiar with this concept, where can i find the definition in the doc. https://github.com/Azure/SONiC/blob/be12cc665c316348352b868f515714f202861f63/doc/kubernetes/Kubernetes-support.md

Reply:
It is the systemctl state of the feature. 
systemctl start marks the feature as "up" and systemctl stop marks it "down".
When we deal with local only, where container start/stop is synchronous, this up/down info is redundant.
But with remote ctr management, the start being asynchronous, there can be a small window, where system state for the feature is "up" but container *not* running yet.

I will update the design doc -- especially in [STATE-DB changes](https://github.com/Azure/SONiC/blob/be12cc665c316348352b868f515714f202861f63/doc/kubernetes/Kubernetes-support.md#feature-status)

Reply:
should this be admin_state?

Reply:
Being all systemctl controlled services, I went for "system_state".  The "admin_state" also can be explained as "administraive state" of the service.

Let me switch to admin state

Reply:
Added section for ["server unreachable"](https://github.com/Azure/SONiC/blob/b0b6080712d0a27c0a4748e921972901a8e3ab65/doc/kubernetes/Kubernetes-support.md#kubernetes-cluster-unreachable-impacts)

Reply:
It is called systemState in CLI too, as it goes in sync with `systemctl start/stop ...`. Would you like me to change in CLI too?

Either we change both or neither, as variable name is preferably in sync with name used in CLI.